addon/components/md-checkbox.js

```diff
@@ -13,22 +14,18 @@ var MdCheckbox = Ember.Component.extend(LayoutRules, RippleMixin, {
 
     attributeBindings: ['isDisabled:disabled', 'aria-label', 'mdNoInk', 'tabindex'],
     tabindex: 0,
+    checked: false,
+
+    didInsertElement() {
+        this._super(...arguments);
+        setupRipples(this, this.$('.md-container'));
```

Comment:
Decided to add some functions to the existing ripples service instead. It's still substantially DRY-ed up compared to the previous implementation